geode-apis-compatible-with-redis/src/main/java/org/apache/geode/redis/internal/data/RedisSortedSet.java

```diff
@@ -401,10 +401,22 @@ long zrevrank(byte[] member) {
     return null;
   }
 
+  List<byte[]> zpopmin(Region<RedisKey, RedisData> region, RedisKey key, int count) {
```

Comment:
Again, we should probably stick to alphabetical order here. Though that means both zpop*'s should go above zscore...

Reply:
Done